uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Test/Fabricator/Contact.php

```diff
@@ -21,4 +21,14 @@ public static function fabricate($params = []) {
 
     return array_shift($result['values']);
   }
+
+  public static function fabricateWithEmail($params, $email) {
+    $contact = self::fabricate($params);
```

Comment:
I'll probably keep this method, For one reason or the other when I pass in the email parameter, the display_name is set to the email irrespective of what ! pass in the display_name parameter or first_name and last_name. Looks like a bug.